xml/System.Windows.Forms/CloseReason.xml

```diff
@@ -149,7 +149,7 @@
       </ReturnValue>
       <MemberValue>4</MemberValue>
       <Docs>
-        <summary>The Microsoft Windows Task Manager is closing the application.</summary>
+        <summary>The Microsoft Windows Task Manager is closing the application.  Because Task Manager in Windows 8.1 and above now issues a "WM_SYSCOMMAND" message with "SC_CLOSE" instead of the "WM_CLOSE" message like Windows 7 and below did, this value is no longer used.  Instead that action is now indistinguishable from and therefore misidentified as "CloseReason.UserClose".</summary>
```

Comment:
```suggestion
        <summary><para>The Microsoft Windows Task Manager is closing the application.</para>
        <para>In Windows 8.1 and later versions, this member is not used, because Task Manager issues the `WM_SYSCOMMAND` message with `SC_CLOSE`. In Windows 7 and earlier versions, the "WM_CLOSE" message was issued. This action is now indistinguishable from `CloseReason.UserClose`.</para></summary>
```

Reply:
Hi @gewarren.  You changed the quoting used on `WM_SYSCOMMAND` and `SC_CLOSE`, but not on `WM_CLOSE`.
I do not know if that quoting style works on the Microsoft docs site, but shouldn't they all be the same?

Reply:
I have no issues with your suggested change, @gewarren.
Should I just commit it, or should someone else review it first?
This is my first pull request for Microsoft docs.

Reply:
You can commit it - thanks @mdell-seradex .

Reply:
Done!